.github/workflows/lint-pr.yml

```
@@ -0,0 +1,20 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
```

I believe we need to trigger on workflow_call:

```suggestion
on:
  workflow_call:
```